boards/st/nucleo_u083rc/doc/index.rst

```diff
@@ -0,0 +1,274 @@
+The Nucleo U083RC board, featuring an ARM Cortex-M0+ based STM32U083RC MCU,
+provides an affordable and flexible way for users to try out new concepts and
+build prototypes by choosing from the various combinations of performance and
+power consumption features. Here are some highlights of the Nucleo U083RC
+board:
+
+
+- STM32U083RC microcontroller in LQFP64 package
+- Two types of extension resources:
+
+  - Arduino Uno V3 connectivity
+  - ST morpho extension pin headers for full access to all STM32U0 I/Os
+
+- On-board STLINK-V2EC debugger/programmer with USB re-enumeration
+  capability: mass storage, Virtual COM port, and debug port
+- Flexible board power supply:
+
+  - USB VBUS or external source(3.3V, 5V, 7 - 12V)
+
+- User LED shared with ARDUINO |reg| Uno V3
+- Two push-buttons: USER and RESET
+- USB Type-C |reg| connector for the ST-LINK
+
+Hardware
+********
+
+The STM32U083xC devices are an ultra-low-power microcontrollers family (STM32U0
+Series) based on the high-performance Arm® Cortex |reg|-M0+ 32-bit RISC core.
+They operate at a frequency of up to 56 MHz.
+
+- Includes ST state-of-the-art patented technology
+- Ultra-low-power with FlexPowerControl:
+
+  - 1.71 V to 3.6 V power supply
+  - -40 °C to +85/125 °C temperature range
+  - 130 nA VBAT mode: supply for RTC, 9 x 32-bit backup registers
+  - 16 nA Shutdown mode (6 wake-up pins)
+  - 30 nA Standby mode (6 wake-up pins) without RTC
+  - 160 nA Standby mode with RTC
+  - 825 nA Stop 2 mode with RTC
+  - 695 nA Stop 2 mode without RTC
+  - 4 µA wake-up from Stop mode
+  - 52 µA/MHz Run mode
+  - Brownout reset
+
+- Core:
+
+  - 32-bit Arm |reg| Cortex |reg|-M0+ CPU, frequency up to 56 MHz
+
+- ART Accelerator:
+
+  - 1-Kbyte instruction cache allowing 0-wait-state execution from flash memory
+
+- Benchmarks:
+
+  - 1.13 DMIPS/MHz (Drystone 2.1)
+  - 134 CoreMark |reg| (2.4 CoreMark/MHz at 56 MHz)
+  - 407 ULPMark™-CP
+  - 143 ULPMark™-PP
+  - 19.7 ULPMark™-CM
+
+- Memories:
+
+  - 256-Kbyte single bank flash memory, proprietary code readout protection
+  - 40-Kbyte SRAM with hardware parity check
+
+- General-purpose input/outputs:
+
+  - Up to 69 fast I/Os, most of them 5 V‑tolerant
+
+- Clock management:
+
+  - 4 to 48 MHz crystal oscillator
+  - 32 kHz crystal oscillator for RTC (LSE)
+  - Internal 16 MHz factory-trimmed RC (±1%)
+  - Internal low-power 32 kHz RC (±5%)
+  - Internal multispeed 100 kHz to 48 MHz oscillator,
+    auto-trimmed by LSE (better than ±0.25 % accuracy)
+  - Internal 48 MHz with clock recovery
+  - PLL for system clock, USB, ADC
+
+- Security:
+
+  - Customer code protection
+  - Robust read out protection (RDP): 3 protection level states
+    and password-based regression (128-bit PSWD)
+  - Hardware protection feature (HDP)
+  - Secure boot
+  - AES: 128/256-bit key encryption hardware accelerator
+  - True random number generation, candidate for NIST SP 800-90B certification
+  - Candidate for Arm® PSA level 1 and SESIP level 3 certifications
```

Comment:
Nit: seeing L56, it seems that registered symbol must be replaced?
```suggestion
  - Candidate for Arm |reg| PSA level 1 and SESIP level 3 certifications
```

Reply:
Done